core/src/main/java/io/kestra/core/runners/NamespaceService.java

```diff
@@ -0,0 +1,15 @@
+package io.kestra.core.runners;
+
+import java.util.*;
+
+public abstract class NamespaceService  {
```

Comment:
As a general rule, it's not common to name a class with utility methods `XxxService`. For this PR, I would not create such a class. The `asTree` method can be added directly to the `NamespaceInterface`.

Reply:
This is just a bring-to-oss operation, this service already exists on EE so I didn't do anything more but ok I'll move that

Reply:
In the end I move it to an Util because I don't want to need an actual Namespace class to use that (as you first retrieve namespaces as String)